prometheus/prom_rules/back_fill/3.8/rules.1.yml

```diff
@@ -153,6 +153,8 @@ groups:
     expr: histogram_quantile(0.95, sum(rate(scylla_storage_proxy_coordinator_cas_write_latency_bucket{}[60s])) by (cluster, le, scheduling_group_name))
     labels:
       by: "cluster"
+  - record: all_scheduling_group
```

Comment:
this is a new recording rules, for the back-filling, it should be under 4.2 (ie. we need a 4.2 directory with that rule in it)